content/blog/2020/03/2020-03-02-pipeline-authoring-sig-update.adoc

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: "Pipeline-Authoring SIG Update"
+tags:
+- Pipeline-Authoring
+- SIG
+- community
+- opensource
+author: markyjackson-taulia
+---
+
+
+==== What is the Pipeline-Authoring Special Interest Group
+
+This special interest group aims to improve and curate the experience of authoring Jenkins Pipelines. This includes the
+syntax of `Jenkinsfile`s and shared libraries, code sharing and reuse, testing of Pipelines and shared libraries, IDE
+integration and other development tools, documentation, best practices, and examples.
+
+=== What Are The Focus Areas of the Pipeline-Authoring Special Interest Group
+
+* Syntax - How `Jenkinsfile`s and shared libraries are written.
+* Code sharing and reuse - Shared libraries and future improvements.
+* Testing - Unit and functional testing of `Jenkinsfile`s and shared libraries.
+* IDE integration, editors, and other development tools - IDE plugins, visual editors, etc.
+* Documentation - Reference documentation, tutorials, and more.
+* Best practices - Defining, maintaining, and evangelizing best practices in Jenkins Pipeline.
+* Examples - Real-world `Jenkinsfile`s and shared libraries demonstrating how to utilize various features of Pipeline,
+as well as basic or starter `Jenkinsfile`s for common patterns that can be used as jumping-off points by new users.
+
+
+=== What Have We Been Up To
+evangelizing
+With the start of a new year, members got together to discuss the roadmap for 2020. During the initial discussions we
+determined that it would be good to examine the goals of previous meetings and determine the best path forward.
+
+A mutual decision was made that to better create a roadmap; we needed to understand better who we were aiming to help.
+We decided that creating personas was very beneficial. Personas are fictional characters, which we are creating based
+upon our research to represent the different user types that might use Jenkins pipelines.
+Creating personas can help us step out of ourselves. It can help us to recognize that different people have different
+needs and expectations, and it can also help us to identify with the user we are building the roadmap for. Personas make
+the task at hand less complicated, they guide our ideation processes, and they can help us to achieve the goal of
+creating a good user experience for our target user group.
+A lot of that work can be found here:
+https://docs.google.com/document/d/1CdyzJwt50Wk3uUNsLMl2d4w2MGYss-phqet0s-KjbEs/edit
+The idea is to map the personas to a maturity model and then map the maturity model to the actual documentation. That
+maturity model can be found here: https://drive.google.com/file/d/1ByzWlPU0j1qM_gqspJppkNKkR5ZVLWlB/view
+
+=== How Can I Get Involved
+
+We have been meeting regularly to define personas to help better us create the SIG roadmap. We meet twice a week,
```

Comment:
```suggestion
We have been meeting regularly to define personas to help us creating a better SIG roadmap. We meet twice a week,
```

Reply:
I actually think both of ours are incorrect now that I read it.
I am away from a computer right now but I will correct this once I am back home